src/fillMachine.js

```diff
@@ -0,0 +1,12 @@
+const fillMachine = (initialStock, totalSlots) => {
+  const inventory = {};
+  const numOfSlots = totalSlots;
+
+  for(let i = 1; i < numOfSlots + 1; i++) {
+    inventory[i] = initialStock[i - 1];
+  }
+
+  return inventory;
```

Comment:
Nice work. I like that you puled this into its own file. The next step is to use `reduce` here.